client/signup/config/flows.js

```diff
@@ -112,6 +112,13 @@ function getDomainSignupFlowDestination( { domainItem, cartItem, siteId, designT
 	return getThankYouNoSiteDestination();
 }
 
+function getEmailSignupFlowDestination( { siteId, siteSlug } ) {
+	return addQueryArgs(
+		{ siteId },
+		`/checkout/thank-you/features/email-license/${ siteSlug }/:receiptId`
+	);
```

Comment:
Keeping `/checkout/thank-you ...` as a destination assumes that users in this flow will always add something to cart. In the screen grab below, I selected a free subdomain and free plan and still got displayed the message "Congratulations on your purchase" when I hadn't purchased anything:

![owef](https://user-images.githubusercontent.com/1269602/188632487-9193212c-0a75-41d5-948b-a35cd5755a80.gif)

Reply:
Same here, #66921 will hide the free domain option in the domain step and #67046 will make adding a mailbox mandatory. There's been a lot of discussion in this PR, so that was probably a bit unclear!